### PR TITLE
Fix jsonrpc client issues

### PR DESCRIPTION
**Pull request by shuo-wu**

### CODE REVIEW

File: pkg/jsonrpc/client.go [hunk 1]
```diff
@@ -44,19 +50,18 @@ func NewClient(conn net.Conn) *Client {
 	return &Client{
 		conn: conn,
 
-		// Get lower 5 digits of the current process ID as the prefix of the message ID
-		idCounter: uint32((os.Getpid() % (1 << 6)) * 10000),
+		idCounter: rand.Uint32() % 10000,
```
Comment (derekbit) [inline pkg/jsonrpc/client.go:53]: Why need a random id?

File: pkg/jsonrpc/client.go [hunk 2]
```diff
@@ -14,7 +15,12 @@ import (
 )
 
 const (
-	DefaultTimeoutInSecond = 30
+	DefaultConcurrentLimit = 1024
+
+	DefaultResponseReadWaitPeriod = 10 * time.Millisecond
+
+	DefaultShortTimeout = 30 * time.Second
+	DefaultLongTimeout  = 24 * time.Hour
```
Comment (derekbit) [inline pkg/jsonrpc/client.go:23]: Can you leave a comment here describing what operations need `DefaultLongTimeout`?

File: pkg/jsonrpc/client.go [hunk 3]
```diff
@@ -181,16 +192,22 @@ func (c *Client) dispatcher() {
...
-			logrus.WithError(err).Errorf("Failed to decoding during read")
-			errChan <- err
-			continue
+			var obj map[string]interface{}
+			if err := decoder.Decode(&obj); err != nil {
+				logrus.WithError(err).Errorf("Failed to decoding during read")
+				errChan <- err
+				continue
+			}
+			c.respReceiverQueue <- obj
```
Comment (derekbit) [inline pkg/jsonrpc/client.go:210]: Any issue in the pervious implementation?

File: pkg/jsonrpc/client.go (hunk 1 above)
Comment (shuo-wu) [reply pkg/jsonrpc/client.go:53]: When there are multiple volumes triggering rebuilding simultaneously, each rebuilding will launch a separate client with the same process id as the 1st ID, then there will be some conflicts.

File: pkg/jsonrpc/client.go (hunk 3 above)
Comment (shuo-wu) [reply pkg/jsonrpc/client.go:210]: I don't know how long `decoder.Decode` would take in average... If it takes hundreds of nanoseconds only, this loop is wasting resources...

**Review by innobead — APPROVED** [review]
LGTM

**Pull request merged**